src/internal/m365/collection/site/backup.go

```diff
@@ -160,9 +160,19 @@ func CollectLists(
 		currPaths = map[string]string{}
 	)
 
+	dps, canUsePreviousBackup, err := parseMetadataCollections(ctx, path.ListsCategory, bpc.MetadataCollections)
+	if err != nil {
+		return nil, false, err
+	}
+
+	// [TODO] utilise deltapaths to determine list's state
```

Comment:
nit: We use `// TODO(hiteshrepo): comment` as a convention for marking todos in code.